README.md

```diff
@@ -37,6 +37,15 @@ TODO: Using the "tree -d" command can be a helpful way to generate this informat
 **{list directories and descriptions}**
 -->
 
+## Development and Software Delivery Lifecycle
```

Comment:
These sections shouldn't be blank. It's a Tier 1 so the software delivery lifecycle shouldn't be too detailed. Local Development can link to contributing.md or something.

Reply:
added text for both of these